### PR TITLE
Adding some regions converted and missing pnach.

### PR DESCRIPTION
**Pull request by backgamon**
All are checked OK on some levels (about 2 hours per game)

### CODE REVIEW

File: patches/SLES_51203_B1252C3D.pnach [hunk 1]
```diff
@@ -0,0 +1,8 @@
+gametitle=Enter the Matrix [PAL-M5] (SLES_512.03) V2.00
+comment=Widescreen hack conversion
+author=Bigdemon
```
Comment (refractionpcsx2) [inline patches/SLES_51203_B1252C3D.pnach:3]: Can you move the comment and author down? :)

File: patches/SLES_55536_E4278493.pnach [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+patch=1,EE,00313360,word,08051d51
+
+/////////////////////////////////////////////
+//patch=1,EE,001475b0,word,3c023f1c //3c023f00
+//Hud Zoom
+//patch=1,EE,003ce03c,word,3c0b3f40 //3c0b3f80
+//Y-Fov
+//patch=1,EE,00112ed4,word,3c023faa //3c023f80
+//patch=1,EE,00112edc,word,3c033cae //3c033c8e
+//patch=1,EE,00112ee0,word,34633000 //3c033c8e
```
Comment (refractionpcsx2) [inline patches/SLES_55536_E4278493.pnach:27]: Is any of this required anymore? if not can you remove it please :)

if it does something else, you can make a separate header for it so its a separate patch, if it's useful

File: patches/SLES-82019_7DE48360.pnach [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+patch=1,EE,0022504c,word,3c033f40 //3c033f80
+
+
+
+
+
+
+ 
+
+
```
Comment (refractionpcsx2) [inline patches/SLES-82019_7DE48360.pnach:22]: can you get rid of some of this whitespace? Thanks :)

**Comment by refractionpcsx2** [comment]
Aside from the whitespace thing, LGTM

**Review by Mrlinkwii — APPROVED** [review]
